### ISSUE TITLE
Why shuffle=False for train_loader?

### ISSUE THREAD

**Issue by bohanzhuang** [jiecaoyu/XNOR-Net-PyTorch#38]
Normally, shuffle should set to be True for train_loader. But why you set it to False instead?  And why use the caffe normalization rather than the default pytorch transformation? Will it have a big influence to the final performance?

**Comment by jiecaoyu** [comment]
@zhuangbohan it can be changed to True.
